client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div>
+    <div
+      v-for="(rule, index) in rules"
+      :key="`rule-${index}`"
+    >
+      <b-row class="justify-content-between">
+        <b-col cols="6">
+          <h5>{{ rule.label }}</h5>
+          <b-form-checkbox
+            v-model="rule.strict"
+            class="mt-3"
+            name="allow"
+            switch
+          >
+            {{ $t('allowRecordsSave') }}
+          </b-form-checkbox>
+        </b-col>
+      </b-row>
+      <div class="mt-3">
+        <b-form-input
+          :placeholder="$t('warningMessage')"
+        />
+        <div class="table-responsive">
+          <table class="table table-borderless">
+            <thead>
+              <tr>
+                <th colspan="2" />
+                <th style="width: 250px;">
+                  {{ $t('valueModifiers') }}
+                </th>
+                <th style="width: 250px;">
+                  {{ $t('multiValues') }}
+                </th>
+              </tr>
+            </thead>
+            <tbody v-if="rule.constraints">
+              <tr
+                v-for="(constraint, consIndex) in rule.constraints"
+                :key="`constraint-${consIndex}`"
+              >
+                <td>{{ constraint.attribute }}</td>
+                <td>{{ constraint.kind }}</td>
+                <td>
+                  <b-form-select
+                    v-model="constraint.modifier"
+                    :options="modifierOptions"
+                    size="sm"
+                  />
+                </td>
+                <td>
+                  <b-form-select
+                    v-model="constraint.multiValue"
+                    :options="multiValueOptions"
+                    :disabled="!constraint.isMulti"
+                    size="sm"
+                  />
+                </td>
+                <td class="text-center">
+                  <c-input-confirm
+                    variant="link"
+                    size="sm"
+                    button-class="text-dark px-0 text-right"
+                    @confirmed="rule.constraints.splice(consIndex, 1)"
+                  />
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
+        <div class="mt-3">
+          <b-row>
+            <b-col
+              sm="12"
+              md="6"
+            >
+              <b-form-group>
+                <b-input-group class="w-100">
+                  <vue-select
```

Comment:
This can be in the table above as the last row, and you can get rid of w-100, and width styles

Reply:
I had to remove it from the table I initially implemented , because the dropdown from the `vue-select` component doesn't display outside the viewport of the table, event with all the positioninig style applied to it